src/scss/grommet-core/_objects.menu.scss

```diff
@@ -49,6 +49,17 @@
 }
 
 .menu__control {
+  &:focus:not(.button--disabled) {
+    box-shadow: inset 0 0 1px 2px $focus-border-color;
```

Comment:
I guess for now this is ok, but we need to decide if we are going to override the default focus for the browser. If the answer is yes, we need to do it for all the elements in grommet. Otherwise we can an outline for one component, and another outline for another.